samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -0,0 +1,197 @@
+
+Models
+======
+
+The following table shows the composition data for the Light Dimmer sample:
+
+.. table::
+   :align: center
+
+   +-------------------------------+
+   | Element 1                     |
+   +===============================+
+   | Config Server                 |
+   +-------------------------------+
+   | Health Server                 |
+   +-------------------------------+
+   | Gen. Level Client             |
+   +-------------------------------+
+   | Gen. OnOff Client             |
+   +-------------------------------+
+
+The models are used for the following purposes:
+
+* The first element contains a Config Server and a Health Server.
+  The Config Server allows configurator devices to configure the node remotely.
+  The Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
+  These callbacks trigger blinking of the LEDs.
+* The third model in the first element is the Generic Level Client.
+  The Generic Level Client controls the Generic Level Server in the target devices, deciding on parameters such as fade time and lighting level.
+* The last model is the Generic OnOff Client which controls the Generic OnOff Server in the target devices, setting the LED state to ON or OFF.
+
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` library to control the buttons on the development kit.
+
+User interface
+**************
+
+Buttons:
+
+ Can be used to input the out-of-band (OOB) authentication value during provisioning. All buttons have the same functionality during this procedure.
+
+Button 1:
+
+ On press and release, Button 1 will publish a Gen. OnOff Set message using the configured publication parameters of its model instance and toggles the LED on/off on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+
+ On press and hold, Button 1 will publishe a Gen. Move Set message using the configured publication parameters of its model instance and will continuously dim the LED lightness state on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+   On release, the button client publish another Gen. Move Set message with the ``delta`` parameter set to 0, making the continuous level change from the previous message stop.
+
+LEDs:
+
+ Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
+
+.. note::
+   :ref:`zephyr:thingy53_nrf5340` supports only one RGB LED.
+   Each RGB LED channel is used as separate LED.
+
+Configuration
+*************
+
+|config|
+
+|nrf5340_mesh_sample_note|
+
+...
+
+Source file setup
+=================
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* A file for handling mesh models, :file:`model_handler.c`.
+
```

Comment:
We need the FEM support section, as long as this sample supports the nRF21540 DK.
```suggestion
FEM support
===========

.. include:: /includes/sample_fem_support.txt

```